datafusion/physical-expr-common/src/aggregate/groups_accumulator/accumulate.rs

```diff
@@ -132,7 +132,7 @@ impl NullState {
         mut value_fn: F,
     ) where
         T: ArrowPrimitiveType + Send,
-        F: FnMut(usize, T::Native) + Send,
+        F: FnMut(usize, bool, T::Native) + Send,
```

Comment:
Can we please update the documentation to reflect this new argument and explains what it means